### DISCUSSION TITLE
SFTP access

### DISCUSSION THREAD

**Discussion by andrejkolesa** [nextcloud/all-in-one#3410] [Questions] [answered]
Hi,

when using Nextclud installed on server, we could use SFTP instead of WEBDAV to conect to it and browse files.

We could connect as: 
sftp://files.domain.dc/remote.php/dav/files/username

But it is not working in AIO, sice 22 port is used for host. 
Is there a way to use some other port (like 8022) and map it to Nextcloud container to port 22 ?

Thank you and br,
Andrey

**Comment by szaimen** [comment, marked as answer]
Hi, Nextcloud does not have built-in SFTP functionality. Only WebDAV is supported so this is not going to work.

However you could of course install a SFTP server on your server directly and connect to that.

**Reply by cielhaidir** [reply]
yes i connect sftp directyl but the files i upload it to the user folder does not show on the ap[

**Comment by ZarVladimirII** [comment]
If someone who is transferring files via SFTP still needs an answer to that 
> yes i connect sftp directyl but the files i upload it to the user folder does not show on the ap[

you have to re-scan with this command  "occ files:scan" until that it will not show up

**Comment by docjyJ** [comment]
Maybe a webdav to sftp proxy? I don't know if it exists...
